src/concurrent_ctx.c

```diff
@@ -13,12 +13,9 @@
 
 static arrayof(redisearch_threadpool) threadpools_g = NULL;
 
-int CONCURRENT_POOL_INDEX = -1;
-int CONCURRENT_POOL_SEARCH = -1;
-
 int ConcurrentSearch_CreatePool(int numThreads) {
   if (!threadpools_g) {
-    threadpools_g = array_new(redisearch_threadpool, 4);
+    threadpools_g = array_new(redisearch_threadpool, 1); // Only used by the coordinator, so 1 is enough
```

Comment:
Whats this pool is used for?

Reply:
the distributed aggregation coordinator thread pool

Reply:
So why we hard code it to 1?